process/block/shardblock.go

```diff
@@ -484,6 +484,11 @@ func (sp *shardProcessor) restoreMetaBlockIntoPool(miniBlockHashes map[string]ui
 			continue
 		}
 
+		processedMiniBlocks := metaBlock.GetMiniBlockHeadersWithDst(sp.shardCoordinator.SelfId())
```

Comment:
Should this happen also on the metaBlock RestoreFromPool?

Reply:
No. Metachain notarizes and processes headers. He does need to look into miniblocks.

Reply:
ok